### PR TITLE
Writing September/October newsletter

### PR DESCRIPTION
**Pull request by gedankenstuecke**
To get the ball rolling, here's the template. What we'd need are news on 

- Public board meeting?
- BOSC (@nlharris would you be willing to write up a few words?)
- Event fellowships (last rounds & upcoming ones)
- GSoC: Do we want to recruit someone to help as an org admin? No one reached out after BOSC afaik

### CODE REVIEW

**Comment by peterjc** [comment]
Does the timing make sense to announce our planned public board meeting for OBF officer elections?

**Comment by gedankenstuecke** [comment]
Yeah, I wondered about that, I guess it depends on how quickly we get this out. Personally I'd be in favour of announcing it through the newsletter too (also gives us a reason to not drag our feet 😉 )

**Comment by nlharris** [comment]
Yes! We should nail down the date for the public Board meeting and announce it in the newsletter, which should be sent out about two weeks before the meeting.
Thank you, Bastian, for volunteering to do the newsletter! I'd be happy to help edit etc.

**Comment by gedankenstuecke** [comment]
Thanks @nlharris! Are you okay for me to use your BOSC report blog post to make a summarized version for the newsletter? 🙂

**Comment by nlharris** [comment]
Yes indeed! In fact, I was going to suggest that. 🙂

BTW, if you're ever short of content, I recently put together this page: https://www.open-bio.org/bosc-dei/

**Comment by gedankenstuecke** [comment]
Thanks amazing, thanks @nlharris!

**Comment by gedankenstuecke** [comment]
I took a stab at it @nlharris

**Comment by nlharris** [comment]
looks good. i have minor suggestions but will wait until the newsletter is more complete before doing an editing pass.

File: newsletters/2023-09.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter September 2023
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting soon; provide a summary of this year's BOSC in Lyonl; share news from the OBF event fellowship; and announce that we are looking for someone to support us in being the organisation-admin for the 2024 Google Summer of Code.
```
Comment (nlharris) [inline newsletters/2023-09.md:11]:
```suggestion
TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
```

File: newsletters/2023-09.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting soon; provide a summary of this year's BOSC in Lyonl; share news from the OBF event fellowship; and announce that we are looking for someone to support us in being the organisation-admin for the 2024 Google Summer of Code. 
+
+In longer form: 
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having it's next public board meeting at XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog.
```
Comment (nlharris) [inline newsletters/2023-09.md:19]:
```suggestion
The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link]. 
```

File: newsletters/2023-09.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having it's next public board meeting at XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog. 
+
+### BOSC news
+
+This year we had the 24th annual Bioinformatics Open Source Conference, which took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceeded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants.
```
Comment (nlharris) [inline newsletters/2023-09.md:23]:
```suggestion
The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
```

File: newsletters/2023-09.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+### Upcoming public board meeting
+
+The OBF is having it's next public board meeting at XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog. 
+
+### BOSC news
+
+This year we had the 24th annual Bioinformatics Open Source Conference, which took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceeded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
+
+In addition to XXX talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_.
```
Comment (nlharris) [inline newsletters/2023-09.md:25]:
```suggestion
In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
```

File: newsletters/2023-09.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+In addition to XXX talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
+
+You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).
+
+### OBF Travel Fellowships
+
+Next round: December 1, 2023
+Last round: August 1, 2023
+
+The ongoing application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
```
Comment (nlharris) [inline newsletters/2023-09.md:34]:
```suggestion
The current application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
```

File: newsletters/2023-09.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+These fellowship can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```
Comment (nlharris) [inline newsletters/2023-09.md:36]:
```suggestion
These fellowships can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```

File: newsletters/2023-09.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+### Google Summer of Code
+
+We are looking for a motivated person who would like to volunteer as the organisation admin to allow the OBF to participate in the 2024 Google Summer of Code. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin.
```
Comment (nlharris) [inline newsletters/2023-09.md:40]:
```suggestion
We are looking for a motivated person who would like to volunteer as the organisation admin to enable the OBF to participate in the 2024 Google Summer of Code, as we have in previous summers. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 
```

File: newsletters/2023-09.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+These fellowship can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
+
+### Google Summer of Code
+
+We are looking for a motivated person who would like to volunteer as the organisation admin to allow the OBF to participate in the 2024 Google Summer of Code. As an organisation we have been providing an umbrella project for various open source bioinformatics projects – including non-member projects – since 2010. In 2023 we sat out one round, as we did not find someone with the time to volunteer as GSoC admin. 
+
+If you are interested to fill that role, please get in touch with us at XXX.
```
Comment (nlharris) [inline newsletters/2023-09.md:42]:
```suggestion
If you are interested in filling that role, please get in touch with us at XXX.
```

File: newsletters/2023-09.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+### ISCBacademy recording
+
+On March 14, the OBF hosted the _ISCBacademy_ where Hannah Wei – co-founder of the Patient-Led Research Collaborative – gave a talk on _"Re-Thinking the Patient’s Role in a Learning Health System: Lessons from the Patient-Led Research Collaborative"_. The [recording of the talk is now on YouTube](https://www.youtube.com/watch?v=M2vAotWKd_Q&feature=youtu.be).
```
Comment (nlharris) [inline newsletters/2023-09.md:46]:
```suggestion
On March 14, 2023, the OBF hosted an _ISCBacademy webinar_ in which Hannah Wei – co-founder of the Patient-Led Research Collaborative – gave a talk on _"Re-Thinking the Patient’s Role in a Learning Health System: Lessons from the Patient-Led Research Collaborative"_. The [recording of the talk is now on YouTube](https://www.youtube.com/watch?v=M2vAotWKd_Q&feature=youtu.be).

Our next ISCBacademy webinar on October 3 will be jointly sponsored with the Bio-Ontologies COSI. Sierra Moxon will speak about "LinkML: an open data modeling framework, grounded with ontologies". Free registration at https://www.iscb.org/iscbacademy.
```

File: newsletters/2023-09.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+## Community updates
+
+### Save the date: CWL Conference 2024
+
+The 2024 CWL Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.
```
Comment (nlharris) [inline newsletters/2023-09.md:52]:
```suggestion
The 2024 Common Workflow Language (CWL) Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.
```

File: newsletters/2023-09.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+## Interesting links
+
+
+
+### Your content needed for the next issue!
+
+Tell us about the things that catch your attention in the open source bioinformatics world! If you have an exciting project update, request for feedback, or interesting link, [feel free to share it with us on GitHub](https://github.com/OBF/newsletter/issues/FIX THIS LINK), or contact Geraldine Van der Auwera (Twitter: [@VdaGeraldine](https://twitter.com/VdaGeraldine), Email: [geraldinevdauwera@gmail.com](mailto:geraldinevdauwera@gmail.com)) if you're not sure whether or not your content is suitable for our newsletter. We're looking for content that's primarily open source / science related.
```
Comment (nlharris) [inline newsletters/2023-09.md:60]: replace Geraldine with Bastian?

**Review by nlharris — CHANGES_REQUESTED** [review]
Thanks, Bastian! I made some minor edits and additions.

**Comment by jiywang3** [comment]
Would you mind add the following (with some modification) into the news letter?

In 2023 BOSC Hackathon, iCn3D viewer (https://www.ncbi.nlm.nih.gov/Structure/icn3d/) got a new feature to show isoforms and exons as tracks. One example is at https://structure.ncbi.nlm.nih.gov/icn3d/share.html?pA3pPu7LxdiuZDVX7: 
image

Users can also predict structures from sequences using ESMFold directly in iCn3D via the menu "File > Predict by Seq. > ESMFold". Other features of iCn3D are listed in its GitHub page: https://github.com/ncbi/icn3d.

**Comment by nlharris** [comment]
Thanks for contributing that, @jiywang3! I think it would be great if the CoFest organizers and participants put together a CoFest 2023 report, with the above and also brief summaries of some of the other CoFest projects. That would be best as a separate document (e.g. a post on the OBF blog), which we could then point to from the newsletter.

I think adding a detailed, illustrated summary of just one of the CoFest projects is out of scope for the OBF newsletter.

File: newsletters/2023-09.md (hunk 11 above)
Comment (gedankenstuecke) [reply newsletters/2023-09.md]: done, thanks!

**Comment by gedankenstuecke** [comment]
Thanks @jiywang3, I've added it as a very brief "project update note" in the respective section. As @nlharris said anything longer would probably be better as its own report on CoFest more broadly (e.g. for the OBF blog!)

**Review by nlharris — APPROVED** [review]

**Comment by gedankenstuecke** [comment]
thanks for the approval @nlharris! I'll be traveling from basically now (sitting at the airport) and won't be fully back online again before the 25th! Which is just to say: If the missing date is fixed while I'm gone it would be amazing if someone else could send it out to not block on me! 🙂

File: newsletters/2023-09.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/FIX THIS LINK)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter September 2023
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code.
```
Comment (nlharris) [inline newsletters/2023-09.md:11]:
```suggestion
TL;DR: In this newsletter we provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
```

File: newsletters/2023-09.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+
+TL;DR: In this newsletter we announce the OBF's upcoming public board meeting; provide a summary of BOSC 2023 in Lyon; share news about the OBF event fellowship; and announce an opening for an OBF admin for the 2024 Google Summer of Code. 
+
+In longer form: 
+
+## OBF News
+
+### Upcoming public board meeting
+
+The OBF is having its next public board meeting on XXX. On our agenda is XXX. You can find more details including how to join us on the OBF Blog [add link].
```
Comment (nlharris) [inline newsletters/2023-09.md:19]:
```suggestion
The OBF will hold its next public board meeting some time this fall (date TBD). When the date is determined, we will announce it on the [OBF Blog](https://www.open-bio.org/blog/) and in social media, as well as email to OBF members.
```

File: newsletters/2023-09.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+### BOSC news
+
+The 24th annual Bioinformatics Open Source Conference (BOSC 2023) took place in July as part of ISMB/ECCB 2023 in Lyon, France, and online. As in prior years, BOSC was preceded by a _CollaborationFest_, a collaborative work event (including but not limited to hacking). This year CoFest was hosted by Jérémy Just at the nearby _École Normale Supérieure de Lyon_, which sponsored space and infrastructure for the 25 in-person participants as well as the numerous online participants. 
+
+In addition to 43 talks and nearly 50 posters, BOSC itself included two keynotes – by Sara El-Gebali, who spoke about _"A New Odyssey: Pioneering the Future of Scientific Progress Through Open Collaboration_ – and by Joseph Yracheta – who spoke about _"The Dissonance between Scientific Altruism & Capitalist Extraction: The Zero Trust and Federated Data Sovereignty Solution”_. The conference closed out with a panel on _Open & Ethical Data Sharing_. 
+
+You can [read the full report about the conference on the OBF blog](https://www.open-bio.org/2023/08/14/bosc-2023-report/).
```
Comment (nlharris) [inline newsletters/2023-09.md:27]: Add:

For those who like to plan ahead, BOSC 2024 will take place between July 12-16, 2024, as part of [ISMB 2024](https://www.iscb.org/ismb2024) in Montréal, Québec, Canada.

File: newsletters/2023-09.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+### OBF Travel Fellowships
```
Comment (nlharris) [inline newsletters/2023-09.md:29]:
```suggestion
### OBF Event Fellowships
```

File: newsletters/2023-09.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+Next round: December 1, 2023
+Last round: August 1, 2023
+
+The current application period for the OBF Event Fellowship lasts until December 1, 2023. So there is still some [time to get your application submitted](https://docs.google.com/forms/d/e/1FAIpQLScMayX_XwTLxc8MiYgQOHIILJNjxF0gL75TUsbaO17h5UuUeA/viewform).
+
+These fellowships can be used to cover registration fees for relevant events related to open source bioinformatics or open science (in-person or virtual), and potentially additional expenses such as travel, lodging, childcare for the duration of the event, small hardware (e.g. a headset or webcam), or high-speed internet. You can read more about [applying for an OBF Event Fellowship](https://www.open-bio.org/event-awards/#fellowships-applications), and [the experiences of past awardees](https://www.open-bio.org/category/travel-fellowship/event-fellowship/).
```
Comment (nlharris) [inline newsletters/2023-09.md:36]: Add

## Sponsors needed!

OBF Event Fellowships, which seek to increase diverse representation at relevant events, are funded by sponsorships. Most of these come through [BOSC](https://www.open-bio.org/events/sponsors/), but we also welcome donations to the OBF. Please contact us at [] if you are interested in discussing sponsorship possibilities.

Comment (gedankenstuecke) [reply newsletters/2023-09.md:38]: Added as well :) Do we have someone that should be the point person to link to?

Comment (nlharris) [reply newsletters/2023-09.md:38]: i don't know. we need someone in OBF to take responsibility for that. (not me!)

Comment (peterjc) [reply newsletters/2023-09.md:38]: Could default to "Please contact the board if ..." (need not be a email link), but I'm inclined to drop this text for now. We don't have anything setup to thank non-BOSC sponsors beyond a one off blog post like we had for the NESCent funds.

Comment (nlharris) [reply newsletters/2023-09.md:38]: ok, we can take this out, but this is something we need to work on.

**Comment by jejust** [comment]
> That would be best as a separate document (e.g. a post on the OBF blog), which we could then point to from the newsletter.
> 
> I think adding a detailed, illustrated summary of just one of the CoFest projects is out of scope for the OBF newsletter.

  I will prepare a draft this week. Even if it's not complete yet, we'll have something to put on the blog and to link from the newsletter.

  BTW, I counted **29 physical attendees** to the CoFest (based on the registration speadsheet, the left-over badges and counting the persons who popped-up during the event without registering). Ten more persons had registered to attend online.

File: newsletters/2023-09.md (hunk 16 above)
Comment (gedankenstuecke) [reply newsletters/2023-09.md:38]: Okay, I'll remove that part!

**Comment by gedankenstuecke** [comment]
That's great, thanks @jejust!

**Comment by nlharris** [comment]
The CoFest 2023 report is now published here: https://www.open-bio.org/2023/09/29/bosc-collaborationfest-2023-report/

**Comment by gedankenstuecke** [comment]
Thanks added in by #36!

File: newsletters/2023-09.md [hunk 17]
```diff
@@ -0,0 +1,71 @@
...
+
+### Save the date: CWL Conference 2024
+
+The 2024 Common Workflow Language (CWL) Conference will be the week of May 13th, 2024 in the Netherlands. More details to follow.
+
+### iCn3D received updates at CoFest
+
+During CoFest 2023, the [iCn3D viewer](https://www.ncbi.nlm.nih.gov/Structure/icn3d/) was updated to include two new features: It can now show isoforms and exons as tracks and users can also predict structures from sequences using ESMFold directly in iCn3D. You can learn more about it [on the project's GitHub page](https://github.com/ncbi/icn3d).
+
+## Interesting links
```
Comment (nlharris) [inline newsletters/2023-09.md:62]: remove this section since it's empty?

**Review by nlharris — APPROVED** [review]
